tests/track/test_track.py

```diff
@@ -15,6 +18,11 @@
 T_MISS_MAX = CONFIG["TRACK"]["IOU"]["T_MISS_MAX"]
 
 
+version.otvision_version = Mock(return_value="ignored")
+version.ottrack_version = Mock(return_value="ignored")
+version.otdet_version = Mock(return_value="ignored")
```

Comment:
Version numbers in output should be ignored at the moment. Otherwise tests will fail on each version update of OTVision.